src/main.rs

```diff
@@ -182,6 +182,15 @@ async fn do_main() -> Result<()> {
 
 	let rpc_url = rpc::check_http(&cfg.full_node_rpc).await?.clone();
 
+	let version = rpc::get_chain(&rpc_url).await?;
+	let runtime_version = rpc::get_runtime_version(&rpc_url).await?;
+	if version != "1.1.0"
```

Comment:
I may be wrong here, but if spec name is not `data-avail` but versions match, then this will not panic, right?

Reply:
yeh it will panic! or in if corrected it will exit